test/test_diffusion.py

```diff
@@ -309,7 +363,8 @@ def sym_eval_mirgecom(expr):
 
             u_mirgecom = sym_eval_mirgecom(p.sym_u)
 
-            diffusion_u_mirgecom = diffusion_operator(discr_mirgecom, alpha=p.alpha,
+            diffusion_u_mirgecom = diffusion_operator(discr_mirgecom,
```

Comment:
Still a bit unhappy with those `_mirgecom` suffixes. `_num`? Something else?

Reply:
The `_mirgecom` suffix is to distinguish from the nodal-dg equivalents (`_ndg` suffix). Both are numerical, so I don't know if I like `_num`... I'm having trouble thinking of something else, apart from maybe just omitting the `_mirgecom` suffix?

Reply:
Ah, duh. Makes sense. I take back what I said.